table/block_based/block_based_table_builder.cc

```diff
@@ -873,7 +873,7 @@ BlockBasedTableBuilder::BlockBasedTableBuilder(
     BlockBasedTable::GenerateCachePrefix<Cache, FSWritableFile>(
         table_options.block_cache_compressed.get(), file->writable_file(),
         &rep_->compressed_cache_key_prefix[0],
-        &rep_->compressed_cache_key_prefix_size);
+        &rep_->compressed_cache_key_prefix_size, "", 0);
```

Comment:
This could be a problem for cache warming. Maybe not in this PR, but we should address it soon. I believe the SST file number is assigned by the CompactionJob when opening the file for writing, so shouldn't be too difficult.

Reply:
Good point! Will add it. I think I missed the cache warming case.